ui/src/menu.tsx

```diff
@@ -58,7 +58,7 @@ export const XMenu = ({ model }: { model: Menu }) => {
         isBeakVisible
         directionalHint={Fluent.DirectionalHint.bottomRightEdge}
         calloutProps={{ styles: { beak: { border: border(1, cssVar('$neutralQuaternaryAlt')) } } }}
-        styles={{ list: { border: border(1, cssVar('$neutralQuaternaryAlt')) } }}
+        styles={{ list: { border: border(1, cssVar('$neutralQuaternaryAlt')), 'button': { lineHeight: '1em'} } }}
```

Comment:
Let's be more specific here and use the class (`.ms-ContextualMenu-link` seems like a good fit) instead of targeting every button component. Why is lineHeight set to `1em`? `unset` seems like a better fit.

```suggestion
        styles={{ list: { border: border(1, cssVar('$neutralQuaternaryAlt')), 'button': { lineHeight: 'unset'} } }}
```